tests/cypress/e2e/fhg544_clearfilters.feature

```diff
@@ -2,9 +2,9 @@ Feature: Filter Local Offer Search Results By Delivery methods Tests
 
 	Scenario: previous servcies are displayed after clearing filters
 	Given a user has arrived on search page
-	When user searches using postcode 'BS2 0SP'
-	Then user can see searvice 'aidforchildrenwithtracheostomies'
+	When user searches using postcode 'E1 2AD'
+	Then user can see searvice 'testservice1'
 	When user filters using language 'Xhosa'
-	Then the user can not see the service 'aidforchildrenwithtracheostomies'
+	Then the user can not see the service 'testservice1'
```

Comment:
please dont add test service1 because it will fail once the data is been removed